gpt_researcher/scraper/pymupdf/pymupdf.py

```
@@ -41,8 +41,16 @@ def scrape(self) -> tuple[str, list[str], str]:
             if self.is_url():
-                response = requests.get(self.link, timeout=5, stream=True)
-                response.raise_for_status()
+                try:
+                    response = requests.get(self.link, timeout=(5, 30), stream=True)
+                    response.raise_for_status()
+                except requests.exceptions.SSLError:
+                    import logging
+                    logging.getLogger(__name__).warning(
+                        f"SSL verification failed for {self.link}, retrying without verification"
+                    )
+                    response = requests.get(self.link, timeout=(5, 30), stream=True, verify=False)
```

Ruff (0.15.5)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)